luigi/contrib/bigquery.py

```diff
@@ -63,6 +64,11 @@ class BQTable(collections.namedtuple('BQTable', 'project_id dataset_id table_id'
     def dataset(self):
         return BQDataset(project_id=self.project_id, dataset_id=self.dataset_id)
 
+    @property
+    def uri(self):
```

Comment:
I would strongly suggest that you add a test for this functionality then. Otherwise it might suddenly be carelessly "optimized away" in some re-factoring.